test/lit/wasm-emscripten-finalize/em_js.wat

```diff
@@ -3,20 +3,32 @@
 
 ;; RUN: wasm-emscripten-finalize %s -S | filecheck %s
 
-;; Both functions should be stripped from the binary
+;; All functions should be stripped from the binary, regardless
+;; of internal name
 ;; CHECK-NOT:  (func
 
+;; The data section that contains only em_js strings should
+;; be stripped.
+;; CHECK-NOT: (i32.const 512) "Only em_js strings here\00")
+
+;; Data sections that also contain other stuff should not be stripped
+;; CHECK: (data (i32.const 1024) "some JS string data\00xxx")
+;; CHECK: (data (i32.const 2048) "more JS string data\00yyy")
```

Comment:
Why aren't the JS strings zeroed out?

Reply:
I've only done the zeroing out thing for the em_asm path so far.     The reason is that I don't currently use the start/end symbols for the em_js path at all.   I think we can do that a followup perhaps, and maybe even completely remove the current 1-export-per-em-js thing.

Reply:
Gotcha. The difference between EM_JS and EM_ASM still confuses me, but that's ok.